mindocr/losses/rec_loss.py

```diff
@@ -47,42 +42,22 @@ def construct(self, pred: Tensor, label: Tensor):
             loss value (Tensor)
         '''
         logit = pred
-        #T, bs, nc = logit.shape
-        #logit = ops.reshape(logit, (T*bs, nc))
         label_values = ops.reshape(label, (-1,))
 
         loss, _ = self.ctc_loss(logit, self.label_indices, label_values, self.sequence_length)
-
-        if self.reduction=='mean':
-            loss = loss.mean()
-
+        loss = self.get_loss(loss)
```

Comment:
is it the same as loss.mean if reduction == 'mean'?

Reply:
If the mode is fp16, then self.get_loss will cast the input to be fp32 first and then calculate the mean, which prevent overflow during sum/mean calculation under fp16. So this is safer.